src/main2.js

```diff
@@ -0,0 +1,272 @@
+window.addEventListener('load', () => {
+    const carousel = document.querySelector('.carousel');
+
+    const buttonLeft = document.createElement('button');
+    buttonLeft.classList.add('button');
+    buttonLeft.classList.add('button--left');
+    buttonLeft.insertAdjacentHTML(
+        'afterbegin',
+        '<svg class="container__svg" style="width:24px;height:24px" viewBox="0 0 24 24"><path class="container__path" fill="#000000" d="M15.41,16.58L10.83,12L15.41,7.41L14,6L8,12L14,18L15.41,16.58Z"/></svg> <p class="screen-readers">Left button for moving photos.</p>'
+    );
+
+    carousel.appendChild(buttonLeft);
+
+    const buttonRight = document.createElement('button');
+    buttonRight.classList.add('button');
+    buttonRight.classList.add('button--right');
+    buttonRight.insertAdjacentHTML(
```

Comment:
dlaczego całego buttona nie stworzysz za pomocą `insertAdjacentHTML`:
const button = `<button class="button button--${side}"><svg>... </button>`
bez sensu dodawać te klasy za pomocą classList.add można to zrobić w stringu